packages/atomic/src/components/result-template-components/atomic-result-icon/atomic-result-icon.pcss

```diff
@@ -0,0 +1,208 @@
+atomic-result-icon {
+    display: block;
+    width: var(--atomic-icon-size);
+    height: var(--atomic-icon-size);
+    background-size: var(--atomic-icon-size) var(--atomic-icon-size);
+    border-radius: 2px;
+}
+
+.salesforce-standard-contact { background-color: rgb(160, 148, 237) }
```

Comment:
I took the following (with some modifications) from the `@salesforce-ux` design system